### ISSUE TITLE
Ignore duplicate entries in the order file

### ISSUE THREAD

**Issue by mjpost** [acl-org/ACLPUB#39]
ACLPUB uses the order file to generate the program order. It checks that all papers are in the file, but if a paper is listed twice, it will generate two copies of it in the proceedings. This should be fixed.

**Comment by rrgerber** [comment]
There was an ACL conference a few years ago that needed to show papers twice (I don't recall why), but in recent years that has not been the case.  If there is a reason why this could happen, then we can change the program and toc generation to only point to a single paper.  But if there is no need to include papers twice, we can just flag an error on it, which would be easier.

**Comment by mjpost** [comment]
I think it's fine to list it 2+ times in the program, but it shouldn't generate two entries under `cdrom/bib` and `cdrom/pdf`, which is what I think happened (right, @bethard?)

**Comment by bethard** [comment]
Yes, ACL 2020 has every paper in two different sessions (so two different times in the schedule). EMNLP 2020 will almost certainly do the same. I would predict that this will be common as many of our conferences go virtual.

And yes, what it does now is generate two entries in cdrom and include the PDF in the full proceedings twice.

**Comment by rrgerber** [comment]
OK, who wants to volunteer to do this?  I know the code pretty well, but don't have time until next week.
